membrane_curvature/base.py

```diff
@@ -0,0 +1,126 @@
+"""
+MembraneCurvature
+=======================================
+
+:Author: Estefania Barreto-Ojeda
+:Year: 2021
+:Copyright: GNU Public License v3
+
+MembraneCurvature calculates the mean and Gaussian curvature of
+surfaces derived from a selection of reference.
+"""
+
+import numpy as np
+import warnings
+from membrane_curvature.surface import get_z_surface
+from membrane_curvature.curvature import mean_curvature, gaussian_curvature
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+import MDAnalysis as mda
```

Comment:
I'd move the general `import MDAnalysis` before the specific one for `AnalysisBase` — it's no big thing but conceptually I like loading the foundation first so if any errors occur then I know that they are in the foundation and not the specifics. (MDAnalysis.analysis is not loaded when you do `import MDAnalysis` so it could conceivably help you in pinning down issues.)